src/content/en/fundamentals/performance/rail.md

```diff
@@ -132,21 +142,16 @@ To evaluate your site against RAIL metrics, use the Chrome DevTools [Timeline to
     <tr>
       <td data-th="RAIL Step"><strong>Response</strong></td>
       <td data-th="Key Metric">Input latency (from tap to paint) < 100ms.</td>
-      <td data-th="User Test">User taps on an icon or button (for example, opening the nav menu, tapping Compose).</td>
+      <td data-th="User Test">User taps a button (for example, opening navigation).</td>
     </tr>
     <tr>
       <td data-th="RAIL Step"><strong>Response</strong></td>
-      <td data-th="Key Metric">Input latency (from tap to paint) < 16ms.</td>
-      <td data-th="User Test">User drags their finger and the app's response is bound to the finger position (for example, pull to refresh, swiping a carousel).</td>
-    </tr>
-    <tr>
-      <td data-th="RAIL Step"><strong>Animation</strong></td>
       <td data-th="Key Metric">Input latency (from tap to paint) < 100ms for initial response.</td>
```

Comment:
I nixed the metric above because it's a confusing example. Is dragging a finger a response or an animation? Maybe we need to add this back, but I need help categorizing and phrasing this.